File: abc/abc221/b.py
# https://atcoder.jp/contests/abc221/submissions/40377697

s = input()
t = input()

l = []
for i in range(len(s)):
  if s[i] != t[i]:
    l.append(i)

if len(l) == 0:
  print('Yes')
elif len(l) == 2:
  if s[l[0]] == t[l[1]] and s[l[1]] == t[l[0]] and not (~l[0] + l[1]): print('Yes')
  else: print('No')
else:
  print('No')

File: abc/abc221/b.cs
// https://atcoder.jp/contests/abc221/submissions/46446198

using System;

class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    string t = Console.ReadLine();

    if(s == t)
    {
      Console.WriteLine("Yes");
      return;
    }

    for(int i = 0; i < s.Length - 1; i++)
    {
      for(int j = i + 1; j < t.Length; j++)
      {
        j = i + 1;
        string u = "";
        for(int k = 0; k < s.Length; k++)
        {
          if(k == i)
          {
            u += s[j];
          }
          else if(k == j)
          {
            u += s[i];
          }
          else
          {
            u += s[k];
          }
        }

        if(t == u)
        {
          Console.WriteLine("Yes");
          return;
        }
        break;
      }
    }

    Console.WriteLine("No");
  }
}

